### ISSUE TITLE
Deadlock on starting make

### ISSUE THREAD

**Issue by kiroma** [termux/termux-app#598]
When I start make the session deadlocks and doesn't respond to any keypresses, not even ^C.

**Comment by Wetitpig** [comment]
Maybe you run multiple builds. I takes time to stop them all.

**Comment by kiroma** [comment]
Nope, only one session started up, nothing appears in terminal when I press keys.

**Comment by kiroma** [comment]
I reinstalled the app from f-droid and it works now.

**Comment by kiroma** [comment]
Not working anymore, even when reinstalled.

**Comment by kiroma** [comment]
`Linux localhost 3.10.61+ #1 SMP PREEMPT Thu Mar 9 00:29:54 CST 2017 aarch64 Android`
Make is the only program experiencing I'm experiencing the issue with.

**Comment by Grimler91** [comment]
Thanks, and are you on android 5, 6, 7 or 8?

**Comment by kiroma** [comment]
5.0.2

**Comment by kiroma** [comment]
Alright I've tried to run it under gdb and it exits without any output with exit code 2.

**Comment by Grimler91** [comment]
@kiroma I've uploaded debug versions of make here: https://grimler.se/dists/testing/debug/binary-aarch64/make_4.2.1_aarch64.deb please install it and run it with gdb and see if it gives more information.

**Comment by kiroma** [comment]
No luck, it just exits.
I tried to record it and reverse step it, but it causes a deadlock even through gdb.
